### PR TITLE
[Google Blockly] Create themes directory

### PR DESCRIPTION
**Pull request by mikeharv**
**Context:** We would like to be able to easily add more themes to Blockly, such as high contrast mode and modes for limited color vision. It would also be great to standardize our approach to themes for Music Lab and our other Google Blockly labs.

**Proposal:** Create a new /themes directory adject to Blockly/addons and move our themes into it. If we want to add new themes, or import them, use this directory as well.

I believe this structure will make it easier to maintain our themes in a way that will be more consistent. 
In each new file, the theme is the default export. If you want to experiment with theme swapping, you can just update the file from which the theme is imported (`GoogleBlocklyWrapper.js` or `MusicBlocklyWorkspace.js`)

*Potential fun follow-up work for the Music Lab crew:* I think that once we move to Thrasos as the default renderer, we can completely deprecate `musicLabDarkTheme` in favor of `cdoDarkTheme` and also remove the custom workspace zoom option.  Happy to discuss the benefits!

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]
I think this organization is great!

File: apps/src/blockly/themes/cdoTheme.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import GoogleBlockly from 'blockly/core';
```
Comment (sanchitmalhotra126) [inline apps/src/blockly/themes/cdoTheme.js]: Should this now be called `cdoLight` (vs. `cdoDark`)?

**Review by sanchitmalhotra126 — APPROVED** [review]
Looks great! Also in favor of using `cdoDark` in place of `musicLabDark` once we have Thrasos (assuming product & design are ok with it too)

File: apps/src/blockly/themes/cdoTheme.js (hunk 1 above)
Comment (mikeharv) [reply apps/src/blockly/themes/cdoTheme.js]: I think keeping it as `cdoTheme` may better suggest that it's the default than `cdoLight` which feels more like a specialization. This isn't a super strong opinion. Thoughts?

Comment (sanchitmalhotra126) [reply apps/src/blockly/themes/cdoTheme.js]: Gotcha, yeah, no strong opinions here either. I could see a case for us wanting to move the notion of what's considered "default" into `googleBlocklyWrapper` though, in which case we could have something like `const DEFAULT_THEME = CdoLight;` in that file?

Comment (mikeharv) [reply apps/src/blockly/themes/cdoTheme.js]: Yeah, that's fair.  If MusicLab is going to be a dark-mode only presentation, we also might need for that one lab to override each theme's component styles. But I think we can cross that bridge when we come to it.

I'm planning to add the other themes soon, so let's table this one potential rename until then?

**Pull request merged**